buildtest/cli/report.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         latest=None,
         oldest=None,
         count=None,
-        pager=None,
+        pagerOpt=None,
```

Comment:
I dont see a point in changing the method signature can you please revert this back. All the arguments are mapping to opts so no point in having `Opt` at end. Can you please fix this throughout code-base.